instantsearch.js/multi-index-hits/src/app.js

```diff
@@ -21,8 +21,9 @@ search.addWidgets([
   instantsearch.widgets.hits({
     container: '#hits-instant-search',
     templates: {
-      item:
-        '{{#helpers.highlight}}{ "attribute": "name" }{{/helpers.highlight}}',
+      item: (hit, { html, components }) => html`
+        ${components.Highlight({ hit, attribute: 'name' })}
```

Comment:
no big deal but really you don't need to put this inside a `html` template. Fine either way for me